cli/running/running.go

```diff
@@ -503,6 +503,12 @@ func FillCtx(cliOpts *config.CliOpts, cmdCtx *cmdcontext.CmdCtx,
 		}
 	}
 
+	if cmdCtx.CommandName != "connect" {
+		if _, err := exec.LookPath(cmdCtx.Cli.TarantoolExecutable); err != nil {
```

Comment:
AFAIK TarantoolExecutable will be empty is it is not found on in configure package. I think we can just check for tarantool executable path emptiness here.

Reply:
Tested, it is empty if not found. Changed.